cypress.json

```diff
@@ -2,5 +2,5 @@
   "video": false,
   "viewportWidth": 1600,
   "viewportHeight": 900,
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://127.0.0.1:3000"
```

Comment:
Is there any reason why using the IP is preferred?

Reply:
Yes, on Windows at least using the IP address speeds up the tests a lot, as it skips the DNS resolving. I had some troubles as some tests were timing out because of this (when Cypress was open with GUI).
When I was running in docker or headless mode it seemed to work just fine.